Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM python:3.10-slim
+
+RUN REQUIRED_PKGS=' \
+        ca-certificates \
+        curl \
+        git \
+        unzip \
```

Comment:
- `ca-certificates`: we shouldn't need to install anymore than what the image includes, unless you're telling me it doesn't have any, then this is acceptable. Have we verified that or is this cargo-culted, is my ask.
- `curl`: doesn't look like we're using cURL anywhere in this Dockerfile, if so, please remove it.
- `git`: all we're using GIt is to download ASDF, for which I'll add comments later.
- `unzip`: same as cURL, we aren't using it so this just bloats the image.

Reply:
Thanks for the review,

- `curl` is used by ASDF to download latest Terraform available version. We can decide to get rid of it or keep it in the image so that the user can eventually update its Terraform version
- `git` is used by both the ASDF installation and the `stacks` tools as well (hosted on GitHub). Good point, this might be removed after
- `unzip` is used by ASDF during when installation Terraform version and also by the installation of the `stacks` tool (mainly by some Python dependencies, but not sure)

For `ca-certificates`, I can't recall, but I think I got TLS warning preventing me to download from GitHub and all Python module dependencies.

For most of them, true, we can decide if we want to keep them or not.

Let me know what you'd prefer.